dev-tools/audius-compose

```diff
@@ -217,7 +217,22 @@ def build(
             f"--project-directory={protocol_dir / 'dev-tools/compose'}",
             f"--file={protocol_dir / 'dev-tools/compose/docker-compose.yml'}",
             f"--project-name={protocol_dir.name}",
-            "--profile=*",
+            "--profile=circle_ci_push",
+            "--profile=tests",
+            "--profile=identity",
+            "--profile=creator",
+            "--profile=discovery",
+            "--profile=comms",
+            "--profile=storagev2",
+            "--profile=notifications",
+            "--profile=elasticsearch",
+            "--profile=mediorum",
+            "--profile=chain",
+            "--profile=poa",
+            "--profile=eth",
+            "--profile=libs",
+            "--profile=solana",
+            "--profile=block-explorer",
```

Comment:
I didn't want to do this, but with the updated docker and compose versions `*` wasn't working anymore